recipes/wpewebkit.recipe

```diff
@@ -67,94 +67,7 @@ class Recipe(recipe.Recipe):
-        'include/wpe-webkit-2.0/wpe/WebKitNavigationAction.h',
-        'include/wpe-webkit-2.0/wpe/WebKitNavigationPolicyDecision.h',
-        'include/wpe-webkit-2.0/wpe/WebKitNetworkProxySettings.h',
-        'include/wpe-webkit-2.0/wpe/WebKitNetworkSession.h',
-        'include/wpe-webkit-2.0/wpe/WebKitNotification.h',
-        'include/wpe-webkit-2.0/wpe/WebKitNotificationPermissionRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitOptionMenu.h',
-        'include/wpe-webkit-2.0/wpe/WebKitOptionMenuItem.h',
-        'include/wpe-webkit-2.0/wpe/WebKitPermissionRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitPermissionStateQuery.h',
-        'include/wpe-webkit-2.0/wpe/WebKitPolicyDecision.h',
-        'include/wpe-webkit-2.0/wpe/WebKitRectangle.h',
-        'include/wpe-webkit-2.0/wpe/WebKitResponsePolicyDecision.h',
-        'include/wpe-webkit-2.0/wpe/WebKitScriptDialog.h',
-        'include/wpe-webkit-2.0/wpe/WebKitScriptWorld.h',
-        'include/wpe-webkit-2.0/wpe/WebKitSecurityManager.h',
-        'include/wpe-webkit-2.0/wpe/WebKitSecurityOrigin.h',
-        'include/wpe-webkit-2.0/wpe/WebKitSettings.h',
-        'include/wpe-webkit-2.0/wpe/WebKitURIRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitURIResponse.h',
-        'include/wpe-webkit-2.0/wpe/WebKitURISchemeRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitURISchemeResponse.h',
-        'include/wpe-webkit-2.0/wpe/WebKitURIUtilities.h',
-        'include/wpe-webkit-2.0/wpe/WebKitUserContentFilterStore.h',
-        'include/wpe-webkit-2.0/wpe/WebKitUserContent.h',
-        'include/wpe-webkit-2.0/wpe/WebKitUserContentManager.h',
-        'include/wpe-webkit-2.0/wpe/WebKitUserMediaPermissionRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitUserMessage.h',
-        'include/wpe-webkit-2.0/wpe/WebKitVersion.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebContext.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebEditor.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebFormManager.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebHitTestResult.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebPage.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebProcessEnumTypes.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebProcessExtension.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebResource.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebsiteDataAccessPermissionRequest.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebsiteData.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebsiteDataManager.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebsitePolicies.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebViewBackend.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebView.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWebViewSessionState.h',
-        'include/wpe-webkit-2.0/wpe/WebKitWindowProperties.h',
-        'include/wpe-webkit-2.0/wpe/webkit.h',
-        'include/wpe-webkit-2.0/wpe/webkit-web-process-extension.h',
+        'include/wpe-webkit-2.0/jsc',
```

Comment:
```suggestion
        'include/wpe-webkit-2.0/jsc',
        'include/wpe-webkit-2.0/wpe',
```

shouldn't it also include `wpe`?

Reply:
Yes, the intention was to use `include/wpe-webkit-2.0` and I was too hasty to push this instead of checking properly. Thanks!